Dockerfile.internal

```diff
@@ -1,4 +1,4 @@
-FROM gradle:6.7.0-jdk11 as build
+FROM ghcr.io/alphagov/verify/gradle:gradle-jdk11 as build
```

Comment:
Shouldn't we still be specifying the gradle version here? If we want to upgrade, we need a managed way to do that, without breaking everything at once?

Reply:
We'd have to change the way we vendor libraries to include a version tag in the vendored images. At the moment, we don't even specify which version of gradle to pull from dockerhub, we just use the tag `jdk11`

https://github.com/alphagov/verify-terraform/blob/e1029473ed20816ad20faa34da4b2f0ed4fe825f/environments/tools/platform/files/pipelines/vendor.yml#L268-L281